.github/workflows/core.yml

```diff
@@ -207,6 +207,10 @@ jobs:
             --extra-index-url https://download.pytorch.org/whl/cpu \
             "$(echo ./firedrake-repo/dist/firedrake-*.tar.gz)[ci,docs]"
 
+          : # DO NOT MERGE
+          pip uninstall -y mpi-pytest
+          pip install "mpi-pytest @ git+https://github.com/firedrakeproject/mpi-pytest.git@connorjward/always-mark"
```

Comment:
I've approved this PR now

Reply:
We use pytest-mpi releases so I will have to make a new release before this goes in. I can do that in my own time though. Thanks for the review!